packages/cli/build.gradle.kts

```diff
@@ -1741,7 +1747,7 @@ graalvmNative {
     builtinCallerFilter = true
     builtinHeuristicFilter = true
     trackReflectionMetadata = true
-    enableExperimentalPredefinedClasses = true
+    enableExperimentalPredefinedClasses = false
```

Comment:
we are disabling this for now, because we don't want `jimage` pre-defined classes to make it into the image at build time